native/cmake/predefine.cmake

```diff
@@ -215,6 +218,8 @@ elseif(ANDROID)
 elseif(OHOS)
     include(${CMAKE_CURRENT_LIST_DIR}/../templates/cmake/ohos.cmake)
 elseif(QNX)
+elseif(NX OR NX_WINDOWS)
```

Comment:
Leave empty for elseif(QNX) ??  Delete the empty block?

Reply:
Because of the addition of error handling, when it is not specified, an error will be reported and the execution will be terminated.
![WXWorkCapture_16525152617142](https://user-images.githubusercontent.com/5205637/168416807-16a40d2a-f4c8-4056-b2d1-3989f4badbbd.png)
Another way is to remove error handling

Reply:
OK, just keep it as it's now.